protocol/testing/genesis.sh

```diff
@@ -22,11 +22,16 @@ DEFAULT_SUBACCOUNT_QUOTE_BALANCE_FAUCET=900000000000000000
 TESTNET_VALIDATOR_NATIVE_TOKEN_BALANCE=1000000000000000000000000 # 1e24
 # Each testnet validator self-delegates 500k whole coins of native token.
 TESTNET_VALIDATOR_SELF_DELEGATE_AMOUNT=500000000000000000000000 # 5e23
+# TODO(GENESIS): 11155111 is the chain ID for sepolia testnet.
```

Comment:
I think lines with `TODO(GENESIS):` are coming from a merge conflict. Can you resolve these?